docs/changelog.rst

```diff
@@ -19,7 +19,7 @@ No changes
 Version 4.1.1
 -------------
 
-* `PR-191 <https://github.com/confluentinc/kafka-connect-elasticsearch/pull/191>`_ - CC-918 Add Kafka Connect Maven plugin to build
+* `PR-191 <https://github.com/confluentinc/kafka-connect-elasticsearch/pull/191>`_ - CC-918 Add |kconnect| Maven plugin to build
```

Comment:
My previous comment was lost. Re-writing here: 

These are mostly autogenerated entries from the PR titles. Adding the aliases instead will require always a second pass (unless it's integrated to the changelog command). And this seems that will incur extra burden. 

We only change the changelog to fix typos or improve the title significantly when needed. 
If feel we should not use aliases here. Just a thought.

Reply:
okay then feel free to ignore